### PR TITLE
Tables for craft skills  (Except Weaving)

### PR DESCRIPTION
**Pull request by B-NUTT**
Technical issues on my end so I'm not going to try and mess with getting this code back on my computer for the misplaced fear of losing it. I haven't had a chance to double-check it because of that.

The only real complication at this point is having recipes use higher-tier/lateral materials if lower tiers are unavailable. At the same time, I really don't care if you can't use it you sell it and there are other solutions and better time syncs rn.

Anyways my code wasn't on my branch and was on master for some reason when I was pushing despite being on my branch the whole time so I was freaked about possibly losing shit. 

It's late mentally this may be unintelligible. And I'm doing Alchemy next because this was a slog and a half doing all the setup. for tables and tools but necessary never the less. So I'm gonna have fun and ball out on the goofy shit. Also if I haven't mentioned there's a lot of inspo being pulled from "survival games" in terms of crafting as it develops just let me know if there's something you want. Also, every single timer and probably amount needs to be changed once crafting skills are done.

TLDR
 couldn't double-check the work but was validated before sending, super sloggy work and many things will need balance with crafting and gathering happening at the same time goofy stuff soon

### CODE REVIEW

**Comment by B-NUTT** [comment]
Piss and also shit

**Comment by B-NUTT** [comment]
Oi vey

**Comment by B-NUTT** [comment]
Listen I've made an honest to god attempt to fix it with juice and nicks help but they are asleep now. I'm not in the headspace to fix it this evening and will not risk breaking it more rn.
